xilem_web/src/message.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{any::Any, fmt::Debug, ops::Deref};
+
+/// A dynamically typed message for the [`View`] trait.
+///
+/// Mostly equivalent to `Box<dyn Any>`, but with support for debug printing.
+// We can't use intra-doc links here because of
+/// The primary interface for this type is [`dyn Message::downcast`](trait.Message.html#method.downcast).
+///
+/// These messages must also be [`Send`].
+/// This makes using this message type in a multithreaded context easier.
+/// If this requirement is causing you issues, feel free to open an issue
```

Comment:
And we should remove this section in both places too

Reply:
Done.